script.bluetooth.delay/addon.py

```diff
@@ -99,24 +79,32 @@ def speakers():
 def main():
 
     if arg == "reset":
-        xbmc.executebuiltin('PlayerControl(stop)')
-        Addon.setSettingBool('state', 1)
-        reset1()
+        if (xbmc.getCondVisibility('Player.HasVideo') == False):
+            xbmc.executebuiltin("PlayMedia("+path+"reset.mp4,1)")
```

Comment:
I'm not sure if we allow including media like this.

Reply:
See [here](https://github.com/xbmc/repo-scripts/pull/2231#discussion_r813231704). This is necessary for resetting the delay value.